Dockerfile

```diff
@@ -1,6 +1,14 @@
 FROM pierremage/hadoop:2.7.2
 MAINTAINER SequenceIQ
 
+# Install jq
+RUN yum -y install wget
+RUN cd /opt \
+      && mkdir jq \
+      && wget -O ./jq/jq http://stedolan.github.io/jq/download/linux64/jq \
```

Comment:
`wget` is not required here. 
I suggest to replace it with `curl -s http://stedolan.github.io/jq/download/linux64/jq > /opt/jq/jq` and delete line 5 `RUN yum -y install wget`.